docs/upstream-differences.md

```diff
@@ -5,7 +5,24 @@ upstream OCP.
 
 ## Installer carry patches
 
-See https://github.com/openshift/installer/compare/release-4.10...jewzaam:release-4.10-azure.
+1. Clone ARO and upstream repos:
+    ```sh
+    # clone our forked installer
+    git clone https://github.com/openshift/installer-aro.git
+    cd installer-aro
+
+    # add the upstream as a remote source
+    git remote add upstream https://github.com/openshift/installer.git
+    git fetch upstream -a
+    ```
+1. See carry patches from previous release:
+    ```sh
+    # list patches
+    git log upstream/release-X.Y-1..origin/release-X.Y-1-azure
```

Comment:
This command lists the carry patches we have put on top of the previous upstream branch, so it has to be X.Y-1 because X.Y is what we will be creating and still does not exist at this point.

The list could be retrieved within github UI in the past because the old repo was a fork. But the new one isn't (from github point of view), so it needs to be done with `git`